carbonmark/components/pages/Project/index.tsx

```diff
@@ -172,12 +175,21 @@ const Page: NextPage<PageProps> = (props) => {
             </div>
           )}
           <div className="descriptionColumn">
-            <Text t="h5" color="lighter">
-              <Trans>Description</Trans>
-            </Text>
-            <Text t="body1">
-              {project.description ?? "No project description found"}
-            </Text>
+            <div className="description">
+              <Text t="h5" color="lighter">
+                <Trans>Description</Trans>
+              </Text>
+              <Text t="body1">
+                {project.description ?? "No project description found"}
+              </Text>
+            </div>
+            {notNil(project.url) && (
+              <Link href={project.url} className="registryLink">
```

Comment:
Link is actually for internal links of the NextJS routing.
For external links there is the `<Anchor>` component. Please compare this again 🙏

Reply:
Good point. Updated.